checkpoint/cloudguard/README.md

```diff
@@ -0,0 +1,167 @@
+#  Check Point Cloudguard agents
+
+## Introduction
+
+This chart creates a single resource management Pod that scans the cluster's resources (Nodes, Images, Pods, Namespaces, Services, PSP, Network Policy, Role, ClusterRole, RoleBinding, ClusterRoleBinding, ServiceAccount, and Ingress) and uploads their meta-data to [Check Point ClougGuard](https://secure.dome9.com/).
+Check Point ClougGuard provides Posture Management, Image Assurance, Visibility, Monitoring and Threat Hunting capabilities.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 3.0+
+- A Check Point ClougGuard account and API key
+
+## Installing the Chart
+
+To install the chart with the chosen release name (e.g. `my-release`), run:
+
+```bash
+$ helm repo add checkpoint-ea https://raw.githubusercontent.com/CheckPointSW/charts/ea/repository/
+$ helm install asset-mgmt checkpoint-ea/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
+```
+
+These are the additional optional flags to enable add-ons:
+
+```bash
+$ 
+$ --set addons.imageScan.enabled=true 
```

Comment:
There are a lot of configuration parameters related to the different add-ons. It would be nice to include a small paragraph that gives a brief description of what each add-on functionality is. It would also help understand the overall capabilities of the solution.